exercises/01-getting-started.md

```diff
@@ -1,63 +1,77 @@
 # Getting started
-In the following we describe how to get started with this project.
+
+In the following, we describe how to get started with this project.
```

Comment:
[pp] This sentence feels redundant - The header is called `getting started`

Reply:
I removed this sentence